<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preview_certificate</class>
 <widget class="QDialog" name="preview_certificate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>250</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>29</y>
        <width>468</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>This is to certify that Mr./Ms __________ a resident of 
_____________ Brgy. San Rafael III, CSJDM, Bulacan, 
SINCE BIRTH his/her rights, including the duties and
Responsibilities accorded by RA 11261 through the 
Oath of Undertaking he/she has signed and 
executed in the Presence of our Barangay Official.</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_certifname">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>s</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_certifaddress">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>s</string>
      </property>
     </widget>
    </widget>
   </item>
   <item alignment="Qt::AlignBottom">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="cert_price">
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btnbox_certificate">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnbox_certificate</sender>
   <signal>accepted()</signal>
   <receiver>preview_certificate</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnbox_certificate</sender>
   <signal>rejected()</signal>
   <receiver>preview_certificate</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
